CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Mason releases
 
+## 0.8.0
+ - Added
+    - zeromq 4.2.2
+    - cppzmq 4.2.1
+
```

Comment:
From conversation in chat: packages have been added in recent release notes. If there are no objections, I would continue with this progress of adding them.